From 632475b20822b010cf18255ce41530ac490cd118 Mon Sep 17 00:00:00 2001
From: Georgi Chorbadzhiyski <gf@unixsol.org>
Date: Fri, 20 Dec 2013 00:25:34 +0200
Subject: [PATCH] Fix compilation error related to removed structure field by
 ffmpeg.

ffmpeg removed reference_dts field in struct AVStream.

Related ffmpeg commit:
  http://git.videolan.org/?p=ffmpeg.git;a=commitdiff;h=2ba68dd044ca8fc591139c05563840f546a9c0c0
---
 lib/xbmc-dll-symbols/DllAvFormat.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/xbmc-dll-symbols/DllAvFormat.c b/lib/xbmc-dll-symbols/DllAvFormat.c
index 4b09663..a592a1f 100644
--- a/lib/xbmc-dll-symbols/DllAvFormat.c
+++ b/lib/xbmc-dll-symbols/DllAvFormat.c
@@ -112,7 +112,9 @@ void xbmc_read_frame_flush(AVFormatContext *s)
 #define RELATIVE_TS_BASE (INT64_MAX - (1LL<<48))
         if(st->first_dts == AV_NOPTS_VALUE) st->cur_dts = RELATIVE_TS_BASE;
         else                                st->cur_dts = AV_NOPTS_VALUE; /* we set the current DTS to an unspecified origin */
+#if LIBAVFORMAT_VERSION_INT < AV_VERSION_INT(55,0,0)
         st->reference_dts = AV_NOPTS_VALUE;
+#endif
 
         st->probe_packets = MAX_PROBE_PACKETS;
 #endif
-- 
1.8.5.5

